app/routes/web/leaderboards.py

```diff
@@ -80,22 +80,30 @@ def resolve_player(
 
     return user
 
-def resolve_mods(score: DBScore) -> int:
+def resolve_mods(score: DBScore, send_nc: bool = True) -> int:
```

Comment:
I think `DBScore` has a `client_version` attribute. Might as well use that if it works. If not, then just leave it :D

Reply:
I need to get version of a client that requests the leaderboards, information about which client submitted the score is useless here

Reply:
Forgot we are not in score submission, my bad.